.github/ISSUE_TEMPLATE/risk-log.md

```diff
@@ -0,0 +1,11 @@
+---
+name: Risk Log
+about: Log the risks here for further discussion
+title: ''
+labels: 'Impact/Severity: Low, Priority: Low, risk, waiting for collaborator review'
```

Comment:
rename the label from `Impact/Severity` to just `Severity`. 
also need to include probability of risk, we only use probability with risks so we can have 
`Risk: unlikely`, `Risk: probable` and `Risk: certain` to tag risks.

Reply:
Rename the tags in bug report/ feature request as well?
I will add unlikely as the default tag for the probability.